app/views/members/_members.uk.html.haml

```diff
@@ -14,13 +14,13 @@
               %span.member-party.inline-post-title
                 - if !member.currently_in_parliament?
                   Former
-                = member.party_name
+                =  member.display_party_name_ukr
```

Comment:
Looks like an extra space has been added before calling 'member'? Would be good to remove that I think :scissors: